diff --git a/tests/data/data1406.c b/tests/data/data1406.c
index 4f9a3fc..078cadf 100644
--- a/tests/data/data1406.c
+++ b/tests/data/data1406.c
@@ -15,7 +15,6 @@ int main(int argc, char *argv[])
   slist1 = curl_slist_append(slist1, "recipient.two@example.com");
 
   curl = curl_easy_init();
-  curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
   curl_easy_setopt(curl, CURLOPT_BUFFERSIZE, 102400L);
   curl_easy_setopt(curl, CURLOPT_URL, "smtp://%HOSTIP:%SMTPPORT/%TESTNUMBER");
   curl_easy_setopt(curl, CURLOPT_USERAGENT, "curl/%VERSION");
