diff --git a/tests/data/data1405.c b/tests/data/data1405.c
index e795508..6bd80cd 100644
--- a/tests/data/data1405.c
+++ b/tests/data/data1405.c
@@ -22,7 +22,6 @@ int main(int argc, char *argv[])
   slist3 = curl_slist_append(slist3, "*FAIL HARD");
 
   curl = curl_easy_init();
-  curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
   curl_easy_setopt(curl, CURLOPT_BUFFERSIZE, 102400L);
   curl_easy_setopt(curl, CURLOPT_URL, "ftp://%HOSTIP:%FTPPORT/%TESTNUMBER");
   curl_easy_setopt(curl, CURLOPT_QUOTE, slist1);
