diff --git a/tests/data/test1402 b/tests/data/test1402
index 6dd6084..e525daf 100644
--- a/tests/data/test1402
+++ b/tests/data/test1402
@@ -33,7 +33,7 @@ http
 --libcurl for simple POST
 </name>
 <setenv>
-SSL_CERT_FILE
+SSL_CERT_FILE=
 </setenv>
 <command>
 http://%HOSTIP:%HTTPPORT/we/want/%TESTNUMBER --libcurl %LOGDIR/test%TESTNUMBER.c -d "foo=bar" -d "baz=quux"
@@ -61,6 +61,8 @@ $_ = '' if /CURLOPT_HTTP_VERSION/
 $_ = '' if /CURLOPT_INTERLEAVEDATA/
 $_ = '' if /CURLOPT_SSLVERSION/
 $_ = '' if /CURLOPT_TIMEOUT_MS/
+$_ = '' if /CURLOPT_VERBOSE/
+$_ = '' if /CURLOPT_CAINFO/
 </stripfile>
 <file name="%LOGDIR/test%TESTNUMBER.c" mode="text">
 %includetext %SRCDIR/data/data%TESTNUMBER.c%
