diff --git a/tests/runtests.pl b/tests/runtests.pl
index 6194961..a9ff39c 100755
--- a/tests/runtests.pl
+++ b/tests/runtests.pl
@@ -568,6 +568,9 @@ sub checksystemfeatures {
                 $pwd = sys_native_current_path();
                 $feature{"win32"} = 1;
             }
+            if($curl =~ /os390|zos|openedition/i) {
+                $feature{"zos"} = 1;
+            }
             if($curl =~ /cygwin|msys/i) {
                 $feature{"cygwin"} = 1;
             }
