diff --git a/src/tool_cfgable.c b/src/tool_cfgable.c
index 3feb3e7..bd5e69f 100644
--- a/src/tool_cfgable.c
+++ b/src/tool_cfgable.c
@@ -160,6 +160,9 @@ static void free_config_fields(struct OperationConfig *config)
   curlx_safefree(config->krblevel);
   curlx_safefree(config->oauth_bearer);
   curlx_safefree(config->sasl_authzid);
+  curlx_safefree(config->upload_encoding);
+  curlx_safefree(config->download_encoding);
+  curlx_safefree(config->upload_encoding_unix);
   curlx_safefree(config->unix_socket_path);
   curlx_safefree(config->writeout);
   curlx_safefree(config->proto_default);
